src/server/supplements/recorder/recorderApp.ts

```diff
@@ -97,11 +97,12 @@ export class RecorderApp extends EventEmitter {
     const recorderPlaywright = require('../../playwright').createPlaywright(true) as import('../../playwright').Playwright;
     const args = [
       '--app=data:text/html,',
-      '--window-size=600,600',
-      '--window-position=1280,10',
+      '--window-size=600,600'
     ];
     if (process.env.PWTEST_RECORDER_PORT)
       args.push(`--remote-debugging-port=${process.env.PWTEST_RECORDER_PORT}`);
+    if (process.env.PWTEST_RECORDER_WINDOW_POSITION)
+      args.push(`--window-position=${process.env.PWTEST_RECORDER_WINDOW_POSITION}`);
```

Comment:
Let's || it with the previous value.

Reply:
I.e. 

```js
args.push(`--window-position=${process.env.PWTEST_RECORDER_WINDOW_POSITION || '1280,10'}`);
```

Reply:
this will change the intent. I'd like to propose to make it work everywhere, and optionally add a way to position the window on different location. That is, the intent is to have this working ok on HDPi (or HD) resolutions laptops without external display attached. Those numbers numbers used for positioning are not the best to use on HDPi screens, especially with different scaling set on system due to high density of the screen (4k or *.k screens).
I understand the codegen is displayed in same location as browser, but at least is controllable using system features (Win+arrow keys on Windows)

Reply:
@pavelfeldman let me know what you folks think about this, I can see the better approach would be via config property,
so it works when run from e.g. VSCode